frontend/packages/operator-lifecycle-manager/src/status/csv-status.ts

```diff
@@ -0,0 +1,62 @@
+import * as _ from 'lodash';
+import { getName } from '@console/shared/src/selectors/common';
+import {
+  ClusterServiceVersionKind,
+  SubscriptionKind,
+  SubscriptionState,
+  ClusterServiceVersionPhase,
+  ClusterServiceVersionStatus,
+} from '../types';
+
+export const subscriptionForCSV = (
+  subscriptions: SubscriptionKind[],
+  csv: ClusterServiceVersionKind,
+): SubscriptionKind =>
+  _.find(subscriptions, {
+    metadata: {
+      namespace: _.get(csv, ['metadata', 'annotations', 'olm.operatorNamespace']),
+    },
+    status: {
+      installedCSV: getName(csv),
+    },
+  } as any); // 'as any' to supress typescript error caused by lodash;
```

Comment:
Can't we use generic type params when calling `_.find` function? Using `as any` is discouraged.